packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -225,7 +224,7 @@ const TagPage = ({classes}: {
             {editing && tag.lesswrongWikiImportSlug && <div className={classes.importNotice}>
               <a target="_blank" rel="noopener noreferrer" href={`http://wiki.lesswrong.com/wiki/${tag.lesswrongWikiImportSlug}`}>See page on old Wiki</a>
               <SeparatorBullet/>
-              {tag.lesswrongWikiImportRevision && 
+              {tag.lesswrongWikiImportRevision &&
```

Comment:
You have to reconfigure your PR to not add all of these spurious changes. It makes PRs much harder to review.

Reply:
Can do. Probably isn't hard.